ADA/databricks_rstudio_sql_warehouse.qmd

```diff
@@ -1,105 +1,210 @@
+```         
+DATABRICKS_HOST=<databricks-host>
+DATABRICKS_SQL_PATH=<sql-warehouse-path>
+DATABRICKS_TOKEN=<personal-access-token> 
+DATABRICKS_CLUSTER_ID=<personal-cluster-id>
+```
 
-    iii) Click install.
+::: callout-important
+Each value should be entered inside quotation marks, like this:
+DATABRICKS_SQL_PATH="/sql/1.0/warehouses/abcdefgh123"
+:::
 
-2.  Get connection details for the SQL Warehouse from Databricks. To set up the connection you will need a few details from a SQL Warehouse within DataBricks.
+Once you have entered the details, save and close your .Renviron file and restart R (Session > Restart R).
 
-    i)  Login to [Databricks](https://adb-6882499576863257.17.azuredatabricks.net/?o=6882499576863257)
+::: callout-note
+Everyone in your team that wishes to connect to the SQL Warehouse in Databricks and run your code must set up their .Renviron file individually, otherwise their connection will fail.
+:::
 
-    ii) Click on the 'SQL Warehouses' tab in the sidebar. ![](../images/databricks-SQL-warehouses.png)
+The sections below describe where to find the information needed for each of the four environment variables.
 
-    iii) Click on the name of the warehouse you want to connect to, and click the 'connection details' tab.
+------------------------------------------------------------------------
 
-    iv) Make a note of the 'Server hostname', 'Port', and 'HTTP Path'.
+#### Databricks host
 
-3.  Get a personal access token from Databricks for authentication.
+------------------------------------------------------------------------
 
-    i)  In Databricks, click on your email address in the top right corner, then click 'User settings'.
+The Databricks host is the instance of Databricks that you want to connect to. It's the URL that you see in your browser bar when you're on the Databricks site and should end in "azuredatabricks.net" (ignore anything after this section of the URL).
 
-    ii) Go to the 'Developer' tab in the side bar. Next to 'Access tokens', click the 'Manage' button. ![](../images/databricks-access-tokens.png)
+------------------------------------------------------------------------
 
-    iii) Click the 'Generate new token' button.
+#### Databricks SQL Path
 
-    iv) Name the token, then click 'Generate'. *Note that access tokens will only last as long as the value for the 'Lifetime (days)' field. After this period the token will expire, and you will need to create a new one to re-authenticate.*
+------------------------------------------------------------------------
 
-    v)  Make a note of the 'Databricks access token' it has given you. **It is important to copy this somewhere as you will not be able to see it through Databricks again.**
+The Databricks SQL Path is the location of the warehouse containing data that you would like to use. To get the warehouse ID, follow these steps:
```

Comment:
SQL Warehouses don't store any data, they only run queries and transport the results to the user/write them to new tables, etc. All data is stored in the unity catalog and can be accessed regardless of the type or compute or the specific SQL warehouse.

Could redraft as "The Databricks SQL warehouse path is the http address of the SQL warehouse you would like to connect to, To get the warehouse ID, follow these steps:"